lib/numbers.md

```diff
@@ -5,20 +5,20 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+Integers, or whole numbers, and floats, or decimals.
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+You can use operations such as +, -, *, /, **, %. Also compare numbers with >, <, =, <=, >=.
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+The '+' operation with numbers will add the numbers togethers to form a total (1+1=2) while with strings it will bring the to strings together, side by side, ('abc' + 'def' = 'abcdef')
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+It will not work because strings are not treated as numbers. You would have to use the .to_i method to take the string and turn it to an integer first.
```

Comment:
I would enter ("20".to_i)/3. The result would be 6 because 3 only goes into 20 6.6667 times. 6 would be the integer.  To get the percise answeer we would have to convert 20 to a float. ("20".to_f)/3 

    On Friday, May 27, 2016 3:41 PM, Alexandra Millatmal <notifications@github.com> wrote:
 

 In lib/numbers.md:>  
>  # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
>  
> -Replace me with your response...
> +It will not work because strings are not treated as numbers. You would have to use the .to_i method to take the string and turn it to an integer first.
Awesome. What if we do so ("20".to_i) and then want to divide the result by 3? Is the result what you would expect?If we wanted the precise answer, would we still convert the string using .to_i?—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.